source/Calamari.Tests/KubernetesFixtures/KubernetesContextScriptWrapperLiveFixtureGke.cs

```diff
@@ -48,7 +48,8 @@ protected override void ExtractVariablesFromTerraformOutput(JObject jsonOutput)
         {
             return new Dictionary<string, string>
             {
-                { "GOOGLE_CLOUD_KEYFILE_JSON", ExternalVariables.Get(ExternalVariable.GoogleCloudJsonKeyfile) }
+                { "GOOGLE_CLOUD_KEYFILE_JSON", ExternalVariables.Get(ExternalVariable.GoogleCloudJsonKeyfile) },
+                { "USE_GKE_GCLOUD_AUTH_PLUGIN", "TRUE"}
```

Comment:
I'm not 100% sure if this is required, according to the [docs ](https://cloud.google.com/blog/products/containers-kubernetes/kubectl-auth-changes-in-gke)you can run into issues. Locally the test doesn't run for me so I'm relying on the pipeline and with the slow feedback cycle decided to include it incase.

Reply:
If we've got a version of `kubectl` that requires the `gke-gcloud-auth-plugin` then it shouldn't be needed; this flag was introduced during the beta/testing releases of these changes to enable customer to opt-in early.

It can't hurt to have it though :)